packages/react/use-overlay/src/OverlayController.tsx

```diff
@@ -16,24 +16,24 @@ export const OverlayController = forwardRef(function OverlayController(
   { overlayElement: OverlayElement, onExit }: Props,
   ref: Ref<OverlayControlRef>
 ) {
-  const [isOpen, setIsOpen] = useState(false);
+  const [isOpenOverlay, setIsOpenOverlay] = useState(false);
```

Comment:
The main reason I revised the naming in OverlayController to be more explicit is for quicker code comprehension by other developers. The terms 'handleClose' and 'isOpen' do not explicitly show what is being closed or opened, which could cause the unfortunate situation where other developers have to read through the code. To prevent this, I've modified it to have a more explicit naming convention.

What do you think? @okinawaa

Reply:
Good! I agree with u!